praw/__init__.py

```diff
@@ -949,6 +950,19 @@ def get_popular_subreddits(self, *args, **kwargs):
         url = self.config['popular_subreddits']
         return self.get_content(url, *args, **kwargs)
 
+    def get_default_subreddits(self, *args, **kwargs):
+        """Return a get_content generator for the default subreddits.
+
+        The additional parameters are passed directly into
+        :meth:`.get_content`. Note: the `url` parameter cannot be altered.
+
+        """
+        url = self.config['default_subreddits']
+        if (len(args) < self.get_content.__code__.co_varnames.index('limit') and
+                'limit' not in kwargs):
+            kwargs['limit'] = 100
```

Comment:
@eleweek You are right that 25 isn't a sensible default for these limits. And I agree having sensible values makes sense. With that said, do you think it would be better to change the default for all the methods?

Either way, that change would be most suitable for a separate pull request. Thus can we use the existing behavior to implement `default_subreddits`?